domain/relation/state/relation.go

```diff
@@ -1734,29 +1653,21 @@ func (st *State) getRelationScope(
 		UUID: uuid,
 	}
 	getScopeStmt, err := st.Prepare(`
-SELECT &getScope.*
-FROM   v_relation_endpoint
-WHERE  relation_uuid = $entityUUID.uuid
-`, relUUID, getScope{})
+SELECT &getScopeID.*
+FROM   relation
+WHERE  uuid = $entityUUID.uuid
```

Comment:
This becomes simplified. Although, I do wonder if we could get rid of it completely with a JOIN?

Reply:
It might be possible, but would be a pretty major refactor, so I elected to leave it as is for now

Reply:
decode has been removed